packages/flutter/lib/src/rendering/custom_paint.dart

```diff
@@ -546,11 +546,6 @@ class RenderCustomPaint extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
-  @override
```

Comment:
You can't just remove the implementation of these methods. As seen by the failing tests, the logic in them is required.

Reply:
Hello @goderbauer 
can you please handle this PR?
because I don't know what to do next for this PR.
Thanks

Reply:
OK. I will close this for now and we will get back to it when we have time.